#3 Implement basic obstecale avoidence.

majawennroth opened the pull request:
This commit makes the car drive forward and stop 15 cm befor hitting an obstecale. solves: #8

majawennroth committed:
#3 Implement basic obstecale avoidence.

This commit makes the car drive forward and stop 15 cm befor hitting an obstecale. solves: #8

platisd commented on car/Obstical_Avoidence/Obstical_Avoidence.ino:
```diff
@@ -0,0 +1,34 @@
+  
+  int distance = front.getDistance();
+
+  if (distance <= 15){ //stop when distance is less than 15 cm.
```
If there is no obstacle, then distance will be `0`. What should your car do then?

platisd commented on car/Blink/Blink.ino:
```diff
@@ -0,0 +1,37 @@
+  delay(1000);                       // wait for a second
+  digitalWrite(LED_BUILTIN, LOW);    // turn the LED off by making the voltage LOW
+  delay(1000);                       // wait for a second
+}
```
Remove this file

majawennroth committed:
#8 Update for no obstecale

makes the car move forward even if there is no obstecale. The blink file is now removed.

majawennroth commented:
@platisd the changes you requested is now in the latest commit.

platisd commented:
> @platisd the changes you requested is now in the latest commit.

There's a little "refresh" icon next to my username. If you press on it a review will be re-requested. :+1:
